src/main/java/ru/javawebinar/topjava/model/Meal.java

```diff
@@ -5,11 +5,14 @@
 import java.time.LocalTime;
 
 public class Meal extends BaseEntity {
-    private final LocalDateTime dateTime;
+    private LocalDateTime dateTime;
```

Comment:
I would recommend to use lombok in models clesses, despite in any case you won't merge current PR with the main branch.

Reply:
When I used @RequiredArgsConstructor I've got several errors like this:
Error:(46, 25) java: constructor Meal in class ru.javawebinar.topjava.model.Meal cannot be applied to given types;
  required: no arguments
  found: java.time.LocalDateTime,java.lang.String,java.lang.Integer
  reason: actual and formal argument lists differ in length

Reply:
Please read more about difference of using @RequiredArgsConstructor, @AllArgsConsturcor & @NoArgsConstructor lombok annotations

Reply:
Do you mean I can use @NoArgsConstructor? But I still must define constructors with parameters of parent class. Also I can use experimental @SuperBuilder annotation. It supports fields from superclasses. But in this case I need to refactor code using builders.

Reply:
I see you have also parent class, so it's not possible to solve it using Lombok. Just as usual define simple constructor which include constructor of parent class